src/main/java/com/develhope/spring/order/service/OrderService.java

```diff
@@ -12,23 +14,38 @@ public class OrderService {
     @Autowired
     private OrderRepository orderRepository;
 
-    public Order insert (Order order) {
+    public Order insert(Order order) {
```

Comment:
Prima di pushare il codice tira un bel pull che ho cambiato un po' di cose in questa classe.

Reply:
O magari fammi la review prima e mergia il mio